arbnode/inbox_tracker.go

```diff
@@ -633,7 +680,7 @@ func (t *InboxTracker) AddSequencerBatches(ctx context.Context, client arbutil.L
 			return errors.New("previous batch accumulator mismatch")
 		}
 
-		if batch.AfterDelayedCount > 0 {
+		if batch.AfterDelayedCount > 0 && !t.snapSyncConfig.Enabled {
```

Comment:
We should read have read delayed message # batch.AfterDelayedCount -1. We should have read any delayed message starting from prevBatchMeta.DelayedMessageCount -1 onward (which will be <= this batch's AfterDelayedCount)

Reply:
done.